src/test/java/mate/academy/validation/PasswordValidatorTest.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.validation;
+
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.model.dto.UserRegistrationDto;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.test.util.ReflectionTestUtils;
+
+class PasswordValidatorTest {
+    private static final String PASSWORD = "123456";
+    private static final String WRONG_REPEAT_PASSWORD = "654321";
```

Comment:
valid passwords in our program have min size of 8
![image](https://user-images.githubusercontent.com/49767049/230966725-5d3a7fd0-e94a-4009-b955-639262f2d1cf.png)